Documentation/CodingGuidelines/CglPhp/CodingBestPractices/Localization.rst

```diff
@@ -20,8 +20,8 @@ Here are some rules to respect when working with labels in :file:`locallang`
 files:
 
 * Always check the existing locallang files to see if a given localized
-  string already exists, in particular :file:`EXT:lang/locallang_common.xlf`
-  and :file:`EXT:lang/locallang_core.xlf`.
+  string already exists, in particular :file:`EXT:core/locallang_common.xlf`
```

Comment:
path should be Resources/Private/Language/locallang_common.xlf and Resources/Private/Language/locallang_core.xlf